src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/Internal/ModelBindingHelper.cs

```diff
@@ -120,41 +85,6 @@ public static class ModelBindingHelper
             params Expression<Func<TModel, object>>[] includeExpressions)
            where TModel : class
         {
-            if (model == null)
```

Comment:
Why remove all this code? We do argument validation in the top level method that user code called.